src/Symfony/Component/Security/Guard/GuardAuthenticatorInterface.php

```diff
@@ -83,6 +83,8 @@ public function getUser($credentials, UserProviderInterface $userProvider);
      *
      * @param mixed         $credentials
      * @param UserInterface $user
+     * 
+     * @return bool
```

Comment:
On a second thought, even if we only care if the method returns true or sth else, encouraging people to return a boolean is a good thing. I'd keep it the way it is now :+1:.